check.go

```diff
@@ -81,18 +81,13 @@ func checkMain(_ *cobra.Command, args []string) error {
 	found := false
 
 	for _, lib := range libs {
-		if lib.LicensePath == "" {
+		if lib.LicenseFile == "" {
```

Comment:
I renamed `LicensePath` to `LicenseFile` so it is clear that we are not talking about a go path here.